aws_test.go

```diff
@@ -64,7 +64,7 @@ func TestAWSRegionCheckLatencyHTTP(t *testing.T) {
 	got := regions[0].GetLatency()
 	want := 15.0
 
-	if got < want || got > want+2 {
```

Comment:
I am not sure why you put +2 here. I ran the test 1000 times and got errors 5 out of 1000 times. I just extended the delay and all tests are working.